Fixture.php

```diff
@@ -20,15 +20,6 @@
  *
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
  */
-abstract class Fixture extends AbstractFixture implements ContainerAwareInterface, DependentFixtureInterface
+abstract class Fixture extends AbstractFixture implements ORMFixtureInterface
```

Comment:
Great change IMO. I believe dependency behavior should be added explicitly, and apart from that it also removes the `EmptyFixture` hacks 👍